bokeh/models/actions.py

```diff
@@ -0,0 +1,17 @@
+""" Client-side interactivity. """
+
+from __future__ import absolute_import
+
+from ..plot_object import PlotObject
+from ..properties import String
+
+class Action(PlotObject):
+    """ Base class for interactive actions. """
+
+class OpenURL(Action):
+    """ Open a URL in a new tab or window (browser dependent). """
+
+    url = String("http://", help="""
+    The URL to direct the web browser to. This can be a template string,
+    which will be formatted with data from the data source.
+    """)
```

Comment:
@birdsarah, to follow up on your removed comment: you can control selection/nonselection glyphs for selection/tap (see `GlyphRenderer`), but not for hover. Support for hover was planned for 0.8, but I'm not sure if I will be able to implement this now (0.9 is more realistic).